solidity/dashboard/package.json

```diff
@@ -45,18 +45,18 @@
     "build": "npm run build-css && react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint src/",
-    "lint:fix": "eslint --fix src/",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
     "format": "npm run lint && prettier --check .",
     "format:fix": "npm run lint:fix && prettier --write ."
   },
   "devDependencies": {
-    "@keep-network/prettier-config-keep": "github:keep-network/prettier-config-keep",
+    "@keep-network/prettier-config-keep": "github:keep-network/prettier-config-keep#a1a333e",
     "@redux-saga/testing-utils": "^1.1.3",
     "@testing-library/react-hooks": "^5.1.2",
     "@types/jest": "^26.0.21",
     "eslint": "^6.8.0",
-    "eslint-config-keep": "git+https://github.com/keep-network/eslint-config-keep.git",
+    "eslint-config-keep": "github:keep-network/eslint-config-keep#0c27ade",
```

Comment:
In some places (like [here](https://github.com/keep-network/tbtc-v2/blob/6f1e52375d1cdcf259ff84af035e5d538730c752/solidity/package.json#L12) we use `#0.3.0` ref. I wonder if there are clear advantages of using one approach over the other, or maybe some conventions of which approach should be used when?
(I'm not requesting a change, I'ts more about educating myself :) )

Reply:
[`0.3.0` tag](https://github.com/keep-network/eslint-config-keep/releases/tag/0.3.0) is older than [0c27ade commit](https://github.com/keep-network/eslint-config-keep/commit/0c27ade54e725f980e971c3d91ea88bab76b2330).

We need to make publishing and tagging of our tools more automatic.